cclib/parser/gaussianparser.py

```diff
@@ -2254,5 +2256,27 @@ def extract_charges_spins(line,prop):
             except:
                 pass
 
+        # Extract Rotational Constants
+        # Example:
+        # Rotational constants (GHZ):           3.13081     1.24272     0.88960
+        # OR for linear molecules:
+        # Rotational constants (GHZ): ************ 12.73690 12.73690
+        # Note: rotational constant will be converted to wavenumber units (1/cm) to standardize across parsers
+        if line[:28] == ' Rotational constants (GHZ):':
+            splits = line[28:].split()
+
+            # Determine if the molecule is linear and only has two constants
+            if '*' in line:  # linear molecule
+                self.append_attribute("rotconsts", [0.0]+[float(splits[i]) for i in (-2, -1)])
```

Comment:
Is there a way with our current test to check this? I was looking at the 'cclib/data/Gaussian/basicGaussian16/CO_TD_delta.log'  and saw that their rotational constants have 0.0000 instead of the * that this parser seems to be looking for:
' Rotational constants (GHZ):           0.0000000          58.2014785          58.2014785'

Reply:
I am not seeing it for HF or CO2 either, using 03, 09, or 16; they all print zero.  I see it for Psi4, and maybe others, so it might have been a holdover from looking at something else.

Reply:
Ok, I've removed that branch in the conditional.